### ISSUE TITLE
`pubs` should not be an error

### ISSUE THREAD

**Issue by benureau** [pubs/pubs#163]
```
$ pubs
error: Too few arguments!

usage: pubs [-h] [--version]
            {init,conf,add,rename,remove,list,edit,tag,statistics,doc,note,export,import,websearch,url,open}
            ...
```
should be:
```
$ pubs
usage: pubs [-h] [--version]
            {init,conf,add,rename,remove,list,edit,tag,statistics,doc,note,export,import,websearch,url,open}
            ...
```
Also, and I have not check if it's present anywhere else, but I feel that the exclamation mark is overly aggressive and unnecessary.

**Comment by benureau** [comment]
Yes, and why I  would be inclined to consider `pubs` as an alias of `pubs --help`, what's crucial to me is to remove the error message.

**Comment by benureau** [comment]
Included in version 0.8.1. Closed by 8fe24140.
